packages/core/src/service/fileUtil.ts

```diff
@@ -25,7 +32,7 @@ export namespace fileUtil {
 		rootUris: readonly RootUriString[],
 	): Generator<string, undefined, unknown> {
 		for (const root of rootUris) {
-			if (uri.startsWith(root)) {
+			if (uriStartsWith(uri, root)) {
 				yield decodeURIComponent(uri.slice(root.length))
```

Comment:
This still causes problems, because the `uri.slice(root.length)` is using the length of an encoded uri to slice a decoded uri, which causes the modules to look like `va/server/...`

Reply:
great point. urghh.